.ci-mgmt.yaml

```diff
@@ -12,17 +12,34 @@ env:
   ARM_TENANT_ID: ${{ secrets.ARM_TENANT_ID }}
   ARM_CLIENT_SECRET: ${{ secrets.ARM_CLIENT_SECRET }}
   PULUMI_MISSING_DOCS_ERROR: true
+  # Would rather fail the build to force listing dependencies manually under plugins: than
+  # auto-install a floating dependency which may influence example generation and cause
+  # non-deterministic builds later on.
+  #
+  # Unfortunately this does not work yet because of e2e tests (example-based tests). Need to find a
+  # way to scope this env var so that it does not configure tests.
+  #
+  #     TestAccFrontdoor: no resource plugin 'pulumi-resource-random' found in the workspace at version v4.15.1 or on your $PATH
+  #     Test_sql/e2e: Error: error resolving type of resource random: internal error loading package "random": no resource plugin
+  #         'pulumi-resource-random' found in the workspace or on your $PATH
+  #
+  # PULUMI_DISABLE_AUTOMATIC_PLUGIN_ACQUISITION: true
```

Comment:
Is it possible to scope this env var to the `tfgen build_sdks` targets in the Makefile?

Reply:
Yes let me look into this in a subsequent PR. This is one round of ci-mgmt edits.